### PR TITLE
Introduce Ansible plugin in Jetbrains section of doc

### PR DESCRIPTION
**Pull request by MSDehghan**
##### SUMMARY
Introduce Ansible plugin in Jetbrains section of doc

##### ISSUE TYPE
- Docs Pull Request

### CODE REVIEW

File: docs/docsite/rst/community/other_tools_and_programs.rst [hunk 1]
```diff
@@ -75,9 +75,9 @@ JetBrains
 
 An open-source Community edition and closed-source Enterprise edition, integrated development environments based on IntelliJ's framework including IDEA, AppCode, CLion, GoLand, PhpStorm, PyCharm and others. Useful JetBrains platform plugins include:
 
+* `Ansible <https://plugins.jetbrains.com/plugin/14893-ansible>`_ - Ansible plugin for Jetbrains IDEs. It provides auto-completion, role name suggestion and other  handy features to work with playbooks and roles.
```
Comment (acozine) [inline docs/docsite/rst/community/other_tools_and_programs.rst]:
```suggestion
* `Ansible <https://plugins.jetbrains.com/plugin/14893-ansible>`_ - general Ansible plugin provides auto-completion, role name suggestion and other handy features for working with playbooks and roles.

```

**Review by acozine — COMMENTED** [review]
Thanks for adding this link to the docs, @MSDehghan! I tightened the wording up just a bit. If you're okay with my edit, please click on "Commit suggestion" on this page. Once the change passes our test suite, I'll merge it.

**Comment by MSDehghan** [comment]
Thank you @acozine .
I committed your suggestion.

**Review by samccann — APPROVED** [review]

**Comment by samccann** [comment]
Thanks @MSDehghan for the Ansible documentation fix!

**Pull request merged**